doc/security/vulnerabilities.rst

```diff
@@ -1358,6 +1358,24 @@ This has been fixed in main for v3.5.0
 - `PR 60079 fix for 3.3
   <https://github.com/zephyrproject-rtos/zephyr/pull/60079>`_
 
+CVE-2023-4264
+-------------
+
+Potential buffer overflow vulnerabilities in the Zephyr Bluetooth subsystem
+
+- `Zephyr project bug tracker GHSA-rgx6-3w4j-gf5j
+  <https://github.com/zephyrproject-rtos/zephyr/security/advisories/GHSA-rgx6-3w4j-gf5j>`_
```

Comment:
it looks like the advisory only captures #60465 in the list of patches? I don't think that's right as we want people digging into the CVE from someplace else (ex. NIST database) to easily be able to access complete info?

Reply:
Yep you are right. I've updated the advisory.